helpers/event/event_slack_notifier.py

```diff
@@ -0,0 +1,11 @@
+import requests
```

Comment:
Make this filename leaner. It is already in a folder called `events`, no need to prefix the filename with `event_`.